src/dune_rules/alias0.ml

```diff
@@ -0,0 +1,16 @@
+open Import
```

Comment:
You could include these in Import.Alias which would save having two potentially confusing modules. If it only gets used by rules, I don't see why not.

Reply:
i.e. inline this entire file into Import and then you can do `Alias.(make check)` etc.